src/store/networkActions.js

```diff
@@ -1,7 +1,10 @@
 import { rpc } from '../lib/rpc';
-import { start } from 'store/store'
+import { start } from '../store/store';
 
 let watchingHeight = false;
+//  (whilei: development: loading so often slows things a lot for me and clutters logs)
+const loadSyncRate = 60 * 1000; // milliseconds
+const loadHeightRate = 5 * 60 * 1000; // ditto
```

Comment:
Not completely related to this PR, but should we define all refresh actions in one place? right now in `store.js` we are refreshing transactions and exchange rates too.